src/translations/client.csv

```diff
@@ -2463,102 +2463,102 @@ v2.constants.week,Label for week,Week,Semaine
-v2.correction.requester.relationshop.intro.label,The title for the corrector form,Note: In the case that the child is now of legal age (18) then only they should be able to request a change to their birth record.,Note: In the case that the child is now of legal age (18) then only they should be able to request a change to their birth record.
+v2.correction.requester.relationshop.intro.label,The title for the corrector form,Note: In the case that the child is now of legal age (18) then only they should be able to request a change to their birth record.,"Note : Si l'enfant a atteint l'âge légal (18 ans), il est le seul à pouvoir demander une modification de son acte de naissance."
 v2.correction.supportingDocuments.attest.label,Label for attestation of supporting documents,Attest,Attestation
 v2.correction.supportingDocuments.notNeeded.label,Label for indicating supporting documents are not needed,Not Needed,Non nécessaire
 v2.custom.field.form.requiredField,,Required for registration,Requis pour l'enregistrement
-v2.error.required,This is the error message for required fields,Required for registration,Required for registration
-v2.error.search,The error message shown when a search query fails,An error occurred while searching,An error occurred while searching
-v2.event.action.declare.form.section.person.field.address.country.label,This is the label for the field,Country,Country
-v2.event.action.declare.form.section.person.field.address.number.label,This is the label for the field,Number,Number
-v2.event.action.declare.form.section.person.field.address.other.addressLine1.label,This is the label for the field,Address Line 1,Address Line 1
-v2.event.action.declare.form.section.person.field.address.other.addressLine2.label,This is the label for the field,Address Line 2,Address Line 2
-v2.event.action.declare.form.section.person.field.address.other.addressLine3.label,This is the label for the field,Address Line 3,Address Line 3
+v2.error.required,This is the error message for required fields,Required for registration,Requis pour l'enregistrement
+v2.error.search,The error message shown when a search query fails,An error occurred while searching,Une erreur s'est produite lors de la recherche
+v2.event.action.declare.form.section.person.field.address.country.label,This is the label for the field,Country,Pays
+v2.event.action.declare.form.section.person.field.address.number.label,This is the label for the field,Number,Numéro
+v2.event.action.declare.form.section.person.field.address.other.addressLine1.label,This is the label for the field,Address Line 1,Adresse ligne 1
+v2.event.action.declare.form.section.person.field.address.other.addressLine2.label,This is the label for the field,Address Line 2,Ligne d'adresse 2
+v2.event.action.declare.form.section.person.field.address.other.addressLine3.label,This is the label for the field,Address Line 3,Ligne d'adresse 3
```

Comment:
In french, Address line 1, Address line 2, ... is shortly *Adresse 1, Adresse 2*.
Could you just replace it by "Adresse 1, Adresse 2, Adresse 3" please?